configure.ac

```diff
@@ -2577,6 +2612,13 @@ AS_IF([test "x$with_systemdsystemunitdir" != "xno"], [
   AC_SUBST([systemdsystemunitdir], [$with_systemdsystemunitdir])
 ])
 
+AC_ARG_WITH([tmpfilesdir],
+  AS_HELP_STRING([--with-tmpfilesdir=DIR], [directory for tmpfiles. See tmpfiles.d(5) for details]),
+  [], [with_tmpfilesdir=no]
+)
+AS_IF([test "x$with_tmpfilesdir" != "xno"], [
+  AC_SUBST([tmpfilesdir], [$with_tmpfilesdir])
+])
```

Comment:
It seems we can ask PKG_CHECK_VAR() for the directory. See  `pkgconf systemd --variable=tmpfilesdir` output. We already use the same for systemdsystemunitdir, see AC_ARG_WITH([systemdsystemunitdir] block in the configure.ac.

Reply:
This will be also usable for uuidd, now in Fedora / RHEL we have uuidd-tmpfiles.conf, I guess we can move it upstream.

Reply:
Wow, that's new for me. I have added it.